src/main/java/com/Assignment72/Controller/LocationController.java

```diff
@@ -1,13 +1,14 @@
 package com.Assignment72.Controller;
 
+import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
 @RestController
 public class LocationController {
     @GetMapping("/locations")//GETの実装
-    public List<LocationResponse> LocationList() {
+    public List<LocationResponse> LocationList(LocationResponse locationResponse) {
```

Comment:
引数なくても問題ないと思うのですがどうでしょう。
```suggestion
    public List<LocationResponse> LocationList() {
```

Reply:
確認、修正いたしました。
c7023ae31dd471327fe6f7cc83ff9a21895e27fd